src/js/utils/getDecodingInfo.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Returns decoding info for a given media configuration.
+ *
+ * @param {MediaDecodingConfiguration} configuration Media configuration.
+ * @returns {Promise<MediaCapabilitiesInfo>} Media decoding information.
+ */
+export default async function getDecodingInfo(configuration) {
+  if (navigator.mediaCapabilities) {
+    return navigator.mediaCapabilities.decodingInfo(configuration);
```

Comment:
The snippet I wrote long time ago would properly fallback when navigator.mediaCapabilities.decodingInfo rejects. See https://developer.chrome.com/blog/media-updates-in-chrome-63-64/#media-capabilities-decoding-info-api

Reply:
getDecodingInfo may reject which could break code at `const decodingInfo = await getDecodingInfo(mediaConfiguration);`
I'd strongly recommend using `isMediaConfigSupported()` at https://developer.chrome.com/blog/media-updates-in-chrome-63-64/#:~:text=isMediaConfigSupported